<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>双码智能识别系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>微软雅黑</family>
       </font>
      </property>
      <property name="title">
       <string>图片列表</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="DragFile" name="listWidget"/>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>上传文件夹</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="startrecon">
            <property name="font">
             <font>
              <family>微软雅黑</family>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>开始检测</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>30</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cleardata">
            <property name="font">
             <font>
              <family>微软雅黑</family>
             </font>
            </property>
            <property name="text">
             <string>清除数据</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Preferred</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="resultlist">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="title">
       <string>识别结果</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QStackedWidget" name="stackwidget">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QTableWidget" name="table_user"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableWidget" name="table_result"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_3">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTableWidget" name="table_analyze"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="show_user">
            <property name="text">
             <string>我的人员</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="show_result">
            <property name="text">
             <string>总体情况</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="show_analyze">
            <property name="text">
             <string>分析结果</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="get_excel">
            <property name="text">
             <string>导出excel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>846</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionchangeuser"/>
    <addaction name="separator"/>
    <addaction name="actionsetdate"/>
    <addaction name="actionaddperson"/>
    <addaction name="separator"/>
    <addaction name="actionclear_data"/>
    <addaction name="actiondele_person"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaddperson">
   <property name="icon">
    <iconset>
     <disabledon>:/res/添加人员.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>添加人员</string>
   </property>
  </action>
  <action name="actionsetdate">
   <property name="icon">
    <iconset>
     <disabledon>:/res/设置时间.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>设置日期</string>
   </property>
  </action>
  <action name="actionconnectdatabase">
   <property name="text">
    <string>连接数据库</string>
   </property>
  </action>
  <action name="actionchangeuser">
   <property name="icon">
    <iconset>
     <disabledon>:/res/切换用户.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>切换用户</string>
   </property>
  </action>
  <action name="actiondele_person">
   <property name="icon">
    <iconset>
     <disabledon>:/res/清空人员.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>清空人员</string>
   </property>
  </action>
  <action name="actionclear_data">
   <property name="icon">
    <iconset>
     <disabledon>:/res/清空记录.png</disabledon>
    </iconset>
   </property>
   <property name="text">
    <string>清空之前结果</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragFile</class>
   <extends>QListWidget</extends>
   <header location="global">dragfile.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
